src/hugo/content/en-us/implementations/aws/software_update/IoT_Jobs/_index.md

```diff
@@ -0,0 +1,156 @@
+---
+
+## Use Cases
+- Performing remote operations on devices connected to IoT Core
+    - _I want to download and install application or firmware updates for remote devices in a fleet_
+    - _I want to apply a security patch or rotate certificates on a remote device_
+    - _I want to reset a device in the field_
+    - _I want to perform remote troubleshooting_
+
+## Reference Architecture
+![IoT Jobs Workflow](IoT_Jobs_Ref_Diagram.png)
+- _AWS IoT Core_ is the MQTT message broker processing messages on behalf of clients
+- _Device_ is the IoT thing to be controlled
+- _Application_ is the remote logic that issues commands
+- _Job_ is the execution of a task on a device
+
+
+1. The _Device_ establishes communication with the _AWS IoT Core_ service and subscribes to the jobs notification topic `$aws/things/thingName/jobs/notify-next`.
+2. A job document is created and stored in S3 bucket. This JSON document contain information that the _Device_ needs to perform a _Job_. It also contains one or more URLs where the _Device_ can download an update or other data.
+3. An _Application_ or user creates a _Job_ in IoT Core to be executed on the device. Data needed for the _Job_ is stored in an S3 bucket.
+4. The IoT Jobs service publishes a message to the jobs topic `$aws/things/thingName/jobs/notify-next` notifying the _Device_ of a pending _Job_.
+5. The _Device_ receives the notification and retrieves the job details from the S3 bucket. 
+6. The _Device_ can update the status of a job execution by publishing status messages to the job update topic `$aws/things/thingName/jobs/JobID/update`. These statuses can include _"IN_PROGRESS"_, _"FAILED"_, _"SUCCEEDED"_ or _"REJECTED"_.
+7. (Optional) Management and monitoring applications can subscribe to certain topics to keep track of the status of jobs. 
+
+## Assumptions
+This implementation approach assumes devices are already provisioned within AWS IoT. Please refer to the [Getting started with AWS IoT Core](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html) for details on connecting your device to AWS IoT. 
+This implementation also assumes that proper permissions have already been granted within IoT Core Policies and S3 resources. 
+
+## Implementation
+
+To create a job, you must first create a job document that is a description of the remote operations to be performed by the devices. A job document contains one or more URLs where the device can download an update or some other data. The job document can be stored in an Amazon S3 bucket, or be included inline with the command that creates the job.
+
+__Sample Job Document__:
+{{< tabs groupId="device-code">}}
```

Comment:
With the job document and python code and separate tabs in the same group, only one can be expanded at a time. I'd suggest getting rid of the job document as a tab, unless you plan on additional examples. If that's the case, select unique `groupId` entries for the documents and code.

Reply:
removed tab for job document